### DISCUSSION TITLE
How to limit access to particular atom?

### DISCUSSION THREAD

**Discussion by headfire94** [reatom/reatom#462] [Q&A]
For example i have module1, atom1 and module2, atom2. 
I don't wan't module2 to have access to atom1. But current library gives access to any atom using `ctx.get` (looking at example on this page https://www.reatom.dev/guides/atomization)

**Comment by BANOnotIT** [comment]
@artalar mentioned nested contexts in chat. I guess these might solve your problem: you define base context and define another one that invested inside the base one and has its own atoms cache.

But overall I would say that limiting access to atom instance itself would be the best option for now.
